lib/base/configobject.cpp

```diff
@@ -565,7 +565,7 @@ void ConfigObject::RestoreObjects(const String& filename, int attributeTypes)
 		if (srs != StatusNewItem)
 			continue;
 
-		upq.Enqueue(std::bind(&ConfigObject::RestoreObject, message, attributeTypes));
+		upq.Enqueue([&message, &attributeTypes](){ RestoreObjects(message, attributeTypes); });
```

Comment:
Explain with your own words why capturing `attributeTypes` by reference is unnecessary.

Reply:
I don't think it's worth capturing an int data type by reference.

Reply:
Exactly. Have a look for more of such in your PR.